eip-5564.md

```diff
@@ -118,49 +120,64 @@ contract IERC5564Messenger {
 }
 ```
 
-An example stealth address scheme for elliptic curves is below. Other stealth schemes may need to modify this approach. **{TODO Modify these steps to match the three function signatures above}**
+An example stealth address scheme for elliptic curves is below. Other stealth schemes may need to modify this approach. 
 
-- $G$ is the generator point of the curve.
+- $G$ represents the generator point of the curve.
 
-- Recipient has private keys $p_{view}$ and $p_{spend}$.
+- Recipient has published a stealth meta-address that constists of the public keys $P_{spend}$ and $P_{view}$
 
-- Recipient publishes corresponding public keys $P_{view}$ and $P_{spend}$ in the `IERC5564Registry`.
+- Recipient has has access to the private keys $p_{spend}$, $p_{view}$ that can access $P_{spend}$ and $P_{view}$.
 
 - Sender generates random 32-byte entropy ephemeral private key $p_{ephemeral}$.
 
-- Sender passes the recipient address and $p_{ephemeral}$ to the `IERC5564Generator` contract's `generateStealthAddress` function.
+- Sender passes the stealth mata-address to the `generateStealthAddress` function.
 
-- This function performs the following computations:
+- The `generateStealthAddress` function performs the following computations:
+  - Generate a random ephemeral private key $p_{ephemeral}$.
```

Comment:
This was already mentioned above, perhaps we delete that bullet (L131) and keep this one?